app/scripts/components/common/banner/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from "react";
+import { Banner as USWDSBanner, BannerContent, Icon } from "@trussworks/react-uswds";
+import SmartLink from '$components/common/smart-link';
+
+const BANNER_KEY = 'dismissedBannerUrl';
+
+function hasExpired(expiryDatetime) {
+  const expiryDate = new Date(expiryDatetime);
+  const currentDate = new Date();
+  return !!(currentDate > expiryDate);
+}
+
+export default function Banner({appTitle, expiryDate, actionUrl, contents}: {appTitle: string, expiryDate: Date, actionUrl: string, contents: string}) {
+  
+  const showBanner = true;//(localStorage.getItem(BANNER_KEY) !== actionUrl);
+  const [isOpen, setIsOpen] = useState(showBanner && !(hasExpired(expiryDate)));
+
+  function onClose () {
+    localStorage.setItem(
+      BANNER_KEY,
+      actionUrl
+    );
+    setIsOpen(false);
+  }
+
+  return (
+    <div>
+      {isOpen && 
+        (<div className='position-relative'>
```

Comment:
I really liked how I don't need any custom styles with built-in utility classes from uswds. However, then we need to add the line here: https://github.com/NASA-IMPACT/veda-ui/pull/1031/files#diff-8890a38014798c861bfe6ddab91bf46ec3c56d330831731a47453f7a1e7bce72R3 which makes the output css huge. (>400kb) So I went back to manual safelists with purgecss: https://github.com/NASA-IMPACT/veda-ui/pull/1031/files#diff-05ed1c7f99e45b485708115502a1e0f28c8547e9a9a29c1b8664f79103cf7873R12

Reply:
Yeah, an alternative to html util classes for more complicated components that works without bloating the CSS would be to use the tokens via mixins, functions or variables in the .scss files. Something like this:

```
// import path to the wanted mixin package (layout mixin, typography mixin, etc)

.some-new-class {
    @include u-padding-x(2);
}
```

But no need to complicate it here:)